db/schema_tables.cc

```diff
@@ -2027,11 +2096,29 @@ std::vector<mutation> make_create_aggregate_mutations(shared_ptr<cql3::functions
     m.set_clustered_cell(ckey, "return_type", aggregate->return_type()->as_cql3_type().to_string(), timestamp);
     m.set_clustered_cell(ckey, "state_func", aggregate->sfunc().name().name, timestamp);
     m.set_clustered_cell(ckey, "state_type", state_type->as_cql3_type().to_string(), timestamp);
-    return {m};
+    std::vector<mutation> muts = {m};
+
+    if (aggregate->has_reducefunc()) {
+        schema_ptr ss = scylla_aggregates();
```

Comment:
why is a schema types variable called ss?

Reply:
Above schema for aggregates is called `s` and its mutation `p`, so I've added prefix 's' for "scylla". I'll make better name for it

Reply:
renamed